packages/core/src/App/Components/Elements/WhatsApp/whatsapp.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Popover, Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import useLiveChat from 'App/Components/Elements/LiveChat/use-livechat';
+import { whatsapp_url } from '@deriv/shared';
+
+type TWhatsAppProps = {
+    can_have_whatsapp: boolean;
+};
+
+const WhatsApp = ({ can_have_whatsapp }: TWhatsAppProps) => {
+    const liveChat = useLiveChat();
+
+    if (!liveChat.isReady) return null;
```

Comment:
[Question]: Why do you need this line of code?

Reply:
CS requirement was to show both WhatsApp and LiveChat icon at same time. We doing the same in deriv.com thats why I created a custom hook to use the same rule for WhatsApp as well 🙏